backend/frontend/views.py

```diff
@@ -76,6 +76,10 @@ def is_super(user):
     return user.is_superuser
 
 
+def custom_permission_denied(request):
```

Comment:
Why is this a function? It's a one line function that gets called once. It looks like it just makes the code harder to read. 

Why not just return the response where it is needed?